xl/plugins.py

```diff
@@ -144,6 +144,8 @@ def uninstall_plugin(self, pluginname):
         return False
 
     def enable_plugin(self, pluginname):
+        if pluginname == 'lyricsviewer':
```

Comment:
Hm. I don't really like having a special case in here. Why not just let it fail?